x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/create_rules_schema.test.ts

```diff
@@ -1066,4 +1066,73 @@ describe('create rules schema', () => {
       }).error
     ).toBeFalsy();
   });
+
+  test('You cannot omit timeline_title when timeline_id are present', () => {
+    expect(
+      createRulesSchema.validate<Partial<RuleAlertParamsRest>>({
+        rule_id: 'rule-1',
+        output_index: '.siem-signals',
+        risk_score: 50,
+        description: 'some description',
+        from: 'now-5m',
+        to: 'now',
+        index: ['index-1'],
+        name: 'some-name',
+        severity: 'severity',
+        interval: '5m',
+        type: 'query',
+        references: ['index-1'],
+        query: 'some query',
+        language: 'kuery',
+        timeline_id: 'some_id',
+        timeline_title: null,
+      }).error
+    ).toBeTruthy();
+  });
+
+  test('You can have timeline_title empty string when timeline_id are present', () => {
+    expect(
+      createRulesSchema.validate<Partial<RuleAlertParamsRest>>({
+        rule_id: 'rule-1',
+        output_index: '.siem-signals',
+        risk_score: 50,
+        description: 'some description',
+        from: 'now-5m',
+        to: 'now',
+        index: ['index-1'],
+        name: 'some-name',
+        severity: 'severity',
+        interval: '5m',
+        type: 'query',
+        references: ['index-1'],
+        query: 'some query',
+        language: 'kuery',
+        timeline_id: 'some_id',
+        timeline_title: '',
+      }).error
+    ).toBeFalsy();
+  });
+
+  test('You cannot have timeline_title whithout timeline_id', () => {
```

Comment:
I could change this title to be: `You cannot have an empty string for timeline_title` and leave the test as is. This is a good test to ensure we protect against accidental empty strings.